### ISSUE TITLE
Express dependency

### ISSUE THREAD

**Issue by alarre** [node-inspector/node-inspector#541]
Hi,

node-inspector depends on Express **"^4.0"**.

However, it really should be **"^4.8.0"** because it uses the method ***ServerResponse.sendFile***, which does not exist in previous versions of Express. For example, I'm using Express 4.6.0 and when I try to start node-inspector I get:

```
TypeError: Object #<ServerResponse> has no method 'sendFile'
   at DebugServer.debugAction (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/node-inspector/lib/debug-server.js:17:7)
   at Layer.handle [as handle_request] (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/layer.js:76:5)
   at next (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/route.js:100:13)
   at Route.dispatch (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/route.js:81:3)
   at Layer.handle [as handle_request] (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/layer.js:76:5)
   at /Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/index.js:227:24
   at Function.proto.process_params (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/index.js:305:12)
   at /Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/index.js:221:12
   at Function.match_layer (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/index.js:288:3)
   at next (/Users/alarre/Documents/nodejs-workspace/borrar/mobile-api/webserver/node_modules/express/lib/router/index.js:182:10)
```

npm install checks if a compatible version of Express is already installed in the "parent" node_modules (which it is: ***4.6.0 ~= ^4.0***) and does not install the correct version under node-inspector/node_modules

Any thoughts?

**Comment by alarre** [comment]
I think the problem was introduced in #514

**Comment by bajtos** [comment]
@alarre could you please submit a pull request fixing the issue?

**Comment by anoopknayak** [comment]
Is this fixed yet?

**Comment by alarre** [comment]
https://github.com/node-inspector/node-inspector/pull/581 should fix it!

**Comment by metaford** [comment]
NAVER - http://www.naver.com/
--------------------------------------------

z.lee@navercorp.com 님께 보내신 메일 <Re: [node-inspector] Express dependency (#541)> 이 다음과 같은 이유로 전송 실패했습니다.

--------------------------------------------

받는 사람이 회원님의 메일을 수신차단 하였습니다. 


--------------------------------------------

**Comment by bajtos** [comment]
Released as `node-inspector@0.9.2`
